tcf_website/templates/search/search.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base/base.html" %}
+{% load static %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'search/search.css' %}" />
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <h1>Search: {{query}}</h1>
+</div>
+
+<hr>
+
+<div class="container">
+    <h4>Courses</h4>
+    {% for c in courses %}
+        <li>
+            <a href="{% url 'course' course_id=c.id %}">{{c.title}} - {{c.number}}</a>
```

Comment:
You can do `{{c.title}} - {{c.code}}` to get the mnemonic in there too

Reply:
the mnemonic isn't indexed currently... i didn't see a field in the `Course` model

Reply:
Oh, maybe in the search view, you can get the course by ID e.g. `Course.objects.get(pk=id)`. that would let you use the `c.code` method

Reply:
done